pico_sdk_version.cmake

```diff
@@ -3,10 +3,18 @@
 set(PICO_SDK_VERSION_MAJOR 1)
 # PICO_BUILD_DEFINE: PICO_SDK_VERSION_MINOR, SDK minor version number, type=int, pico_base
 # PICO_CONFIG: PICO_SDK_VERSION_MINOR, SDK minor version number, type=int, pico_base
-set(PICO_SDK_VERSION_MINOR 1)
+set(PICO_SDK_VERSION_MINOR 2)
 # PICO_BUILD_DEFINE: PICO_SDK_VERSION_REVISION, SDK version revision, type=int, pico_base
 # PICO_CONFIG: PICO_SDK_VERSION_REVISION, SDK version revision, type=int, pico_base
 set(PICO_SDK_VERSION_REVISION 0)
+# PICO_BUILD_DEFINE: PICO_SDK_VERSION_PRE_RELEASE_ID, optional SDK pre-release version identifier, type=string, pico_base
+# PICO_CONFIG: PICO_SDK_VERSION_PRE_RELEASE_ID, SDK version revision, type=string, pico_base
+set(PICO_SDK_VERSION_PRE_RELEASE_ID develop)
```

Comment:
Does this also get displayed by the version-number in the Doxygen output?

Reply:
yes